### PR TITLE
Add Lucidchart, Basecamp, Miro, and Beautiful.ai

### PR DESCRIPTION
**Pull request by SEM-DEV**
Suitable software for productivity.

### CODE REVIEW

**Comment by AchoArnold** [comment]
Hello @SEM-DEV 

`(https://basecamp.com/discounts) [FREE]` is not free it should be DISCOUNT since basecamp gives a 10% discount to students.

File: README.md [hunk 1]
```diff
@@ -167,6 +167,10 @@ Table of Contents
 
 * [Tabula](https://docs.tabula.io/getting-started/plans-and-subscriptions/tabula-for-education) [DISCOUNT] - Tabula.io is a modern no-code data analytics and visualisation tool. Students get 60% permanent discount on Tabula Solo plan.
 * [Craft](https://www.craft.do/education) [FREE] - craft.do is an app for taking notes, writing to-do lists, organizing important documents, organizing them.
+* [Lucidchart](https://www.lucidchart.com/pages/usecase/education) [FREE] - Create flowcharts & diagrams online with Lucidchart. You don't need to download anything. Students are eligible for a free Education account using school email.
+* [Basecamp](https://basecamp.com/discounts) [FREE] - Basecamp's the project management platform that helps small teams move faster and make more progress than they ever thought possible. Sign up using school email and follow the steps from the link above.
```
Comment (AchoArnold) [inline README.md:171]: Discount

**Review by AchoArnold — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SEM-DEV) [reply README.md:171]: Do can see this below:
**Free for teachers and students**, 10% off for non-profits and charities. [30-day free trial](https://basecamp.com/pricing) for everyone (no credit card required).

Comment (SEM-DEV) [reply README.md:171]: It is totally free for teachers and students.

Comment (AchoArnold) [reply README.md:171]: My bad, I didn't read it properly.

**Pull request merged**